region/lowernorfair/west/Fast Ripper Room.json

```diff
@@ -118,9 +118,49 @@
         "h_heatProof",
```

Comment:
There's a potential for net energy gain here, so we can drop the `h_heatProof` requirement and add `heatFrames` instead. Then the randomizer will be able to model gaining arbitrary amounts of energy: with the frame counts written here, without heat protection a net energy gain would only be logical on Expert and higher settings, which seems right.